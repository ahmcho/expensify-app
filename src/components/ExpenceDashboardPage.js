import React from 'react';
import ExpenseList from './ExpenseList';
import ExpenseListFilters from  './ExpenseListFilters';

const ExpenceDashboardPage = () => (
	<div>
		<ExpenseListFilters />
		<ExpenseList />
	</div>
)



export default ExpenceDashboardPage;