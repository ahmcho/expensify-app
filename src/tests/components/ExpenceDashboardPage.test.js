import {shallow} from 'enzyme';
import React from 'react';
import ExpenceDashboardPage from '../../components/ExpenceDashboardPage';

test('Should render ExpenceDashboardPage correctly', () => {
    const wrapper = shallow(<ExpenceDashboardPage />);
    expect(wrapper).toMatchSnapshot();
})