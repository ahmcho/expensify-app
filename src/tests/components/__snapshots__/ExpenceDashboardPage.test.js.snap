// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render ExpenceDashboardPage correctly 1`] = `
<div>
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
